library/lib/node-utils.js

```diff
@@ -49,33 +25,15 @@ async function evalInFork(source, map) {
         else resolve(messageData.join(''))
       } else reject(new Error(errData.join('')))
     })
-    js.send(source)
-    js.send(map)
-  })
-}
-
 
-function withRawErrorStack(fn) {
-  const $prepareStackTrace = Error.prepareStackTrace
-  Error.prepareStackTrace = (error, stack) => stack
-  try { return fn() } finally { Error.prepareStackTrace = $prepareStackTrace }
+    js.send(appendSourceMap(name, source, map))
+  })
 }
 
-function toMappedStack(createMap, stack = [], { evalOnly = false } = {}) {
-  const sourceMap = new SourceMapConsumer(createMap())
-  return stack
-    .map(frame => {
-      if (evalOnly && !frame.isEval()) return null
+function appendSourceMap(name, source, map) {
+  map.sources = map.sources.map(source => { try { return require.resolve(source) } catch (_) { return `/.../${source}` } })
```

Comment:
Wat beteken 3 dots hier precies? Ik kan er niet zoveel over vinden behalve dat het in bash hetzelfde is als `../..`

Reply:
Niets, het enige wat ik wil zeggen is dat ik niet weet waar de code staat, waarschijnlijk omdat het gegenereerde code is. Ik kon even niets anders bedenken. Dingen als `generated` en `unknown` gaven meer onduidelijkheid. Het moet sowieso met een `/` beginnen omdat er anders meer onduidelijkheid onstaat.

Reply:
Zou zelf tevreden zijn met alleen de `source` dan denk ik. Denk dat als je het misschien wil kopiëren je dan sneller er achter komt dat het nergens opslaat dan met de puntjes

Reply:
@larsvankleef Dat kan niet omdat die vaak niet absoluut zijn en dan wordt ie geprefixed met de `sourceRoot` en dan staat er dus iets als `/home/eecolor/kaliberjs/build/example/.kaliber-eval/webpack/bootstrap`. Om dat te voorkomen moet ie beginnen met een `/`, ik heb `/generated:webpack/bootstrap`, `/webpack/bootstrap`, `/unknown:webpack/bootstrap` en dergelijke geprobeerd. `/.../webpack/bootstrap` was het minst verwarrend.

Uiteindelijk verwacht ik dat iedereen het simpelweg negeert (behalve ik).